src/wcc.js

```diff
@@ -105,10 +104,11 @@ async function getTagName(moduleURL) {
 async function initializeCustomElement(elementURL, tagName, attrs = []) {
   await registerDependencies(elementURL);
 
+  const { pathname } = elementURL;
```

Comment:
Avoiding direct use of `import(elementURL)` since when converted to CJS, `require` will break when given a `URL`.